added 2 new snap sync pages (#525)
ethereum-optimism/docs: pages/builders/chain-operators/management/snap-sync.mdx

---
title: Using Snap Sync
lang: en-US
description: Learn how to use snap sync on your OP chain.
---

import { Callout, Steps } from 'nextra/components'

# Using Snap Sync

> Need brief description of what snap sync is + why its important for chain operators.

Reviewer: Snap sync enables users to more quickly sync their nodes & is a mechanism for node operators to sync chains that are using blobs without a blob archiver.

## Enable Snap Sync for Chains

> This is for the run book for chain operators

Reviewer: Chain operators need to spin up a node which is exposed to the network & has tx gossip disabled.

---
title: Using Snap Sync
lang: en-US
description: Learn how to use snap sync on your OP chain.
---

import { Callout, Steps } from 'nextra/components'

# Using Snap Sync

> 

Reviewer 2: I think this page would benefit from the same copy in the node section:

> This guide reviews the optional feature of Snap Sync for node operators, including benefits and how to enable the feature.

> Snap Sync significantly improves the experience of syncing an OP Stack node. Snap Sync is a native feature of go-ethereum that is now optionally enabled on op-node & op-geth. Snap Sync works by downloading a snapshot of the state from other nodes on the network and is then able to start executing blocks from the completed state rather than having to re-execute every single block. This means that performing a Snap Sync is significantly faster than performing a full sync.

Author: done

> Snap sync enables developers to more quickly sync their nodes and is a mechanism for node operators to sync chains that are using blobs without a blob archiver.

Reviewer 2: Suggestion, turn this into a bulleted list:
- Snap sync enables node operators on your network to sync faster.
- It removes the need for nodes on your post Ecotone network to run a [blob archiver](/builders/node-operators/management/blobs)

<Callout type="warning">
  For snap sync, all `op-geth` nodes should be ran with public IP addresses to easily find other op-geth nodes to sync from. 
  The only exception is for sequencers and transaction ingress nodes.
</Callout>

## Enable Snap Sync for Chains

> 

Reviewer 2: I think this section could use the nextra steps component with two steps:
1. Setup snap sync node in your infrastructure
2. Tell your network they can use snap sync

Author: thx for this but def need more context here -- i.e., how exactly does a chain operator complete these steps? where/how do they make the configuration changes?

> To enable snap sync, chain operators need to spin up a node which is exposed to the network and has transaction gossip disabled.

Reviewer 2: This would be step 1: setup a snap sync node
Sub steps:
- Expose `op-geth` discovery port to the internet, need to figure out the default port and if there's any conflict with `geth`
- Disable transaction gossip with: /builders/node-operators/management/configuration#rollupdisabletxpoolgossip

Reviewer 2: /builders/node-operators/management/configuration#discoveryport

Author: got it! thank you! 😄

> 

Reviewer 2: Step 2: inform node operators on your network they can utilize snap sync
Sub steps:
- point to the node operator section for snap sync

Author: done

lang: en-US
description: Learn how to use and enable snap sync on your OP chain.
---

import { Callout, Steps } from 'nextra/components'

# Using Snap Sync for Chain Operators

This guide reviews the optional feature of Snap Sync for OP chains, including benefits and how to enable the feature.

Snap Sync significantly improves the experience of syncing an OP Stack node. Snap Sync is a native feature of go-ethereum that is now optionally enabled on `op-node` & `op-geth`.
Snap Sync works by downloading a snapshot of the state from other nodes on the network and is then able to start executing blocks from the completed state rather than having to re-execute every single block.
This means that performing a Snap Sync is significantly faster than performing a full sync.

*   Snap sync enables node operators on your network to sync faster.
*   Snap sync removes the need for nodes on your post Ecotone network to run a [blob archiver](/builders/node-operators/management/blobs).

## Enable Snap Sync for Chains

To enable snap sync, chain operators need to spin up a node which is exposed to the network and has transaction gossip disabled.

<Callout type="warning">

>   For snap sync, all `op-geth` nodes should be ran with the [`--discovery.port`](/builders/node-operators/management/configuration#discoveryport) exposed to easily find other op-geth nodes to sync from.

Reviewer: This callout is a little misleading. We should be instructing people to expose port 30303 TCP and 30303 UDP. If they set the port with `--disovery.port` they need to open the port they specified for UDP. If they set `--port`, they need to open up the port that the specified for TCP.

Author: updated

---
title: Using Snap Sync for Chain Operators
lang: en-US
description: Learn how to use and enable snap sync on your OP chain.
---

import { Callout, Steps } from 'nextra/components'

# Using Snap Sync for Chain Operators

This guide reviews the optional feature of Snap Sync for OP chains, including benefits and how to enable the feature.

Snap Sync significantly improves the experience of syncing an OP Stack node. Snap Sync is a native feature of go-ethereum that is now optionally enabled on `op-node` & `op-geth`.
Snap Sync works by downloading a snapshot of the state from other nodes on the network and is then able to start executing blocks from the completed state rather than having to re-execute every single block.
This means that performing a Snap Sync is significantly faster than performing a full sync.

*   Snap sync enables node operators on your network to sync faster.
*   Snap sync removes the need for nodes on your post Ecotone network to run a [blob archiver](/builders/node-operators/management/blobs).

## Enable Snap Sync for Chains

To enable snap sync, chain operators need to spin up a node which is exposed to the network and has transaction gossip disabled.

> <Callout type="warning">

Reviewer: ```suggestion
<Callout type="info">
```

For snap sync, all `op-geth` nodes should expose port `30303` TCP and `30303` UDP to easily find other op-geth nodes to sync from. 
  *   If you set the port with [`--discovery.port`](/builders/node-operators/management/configuration#discoveryport), then you must open the port specified for UDP. 
  *   If you set [`--port`](/builders/node-operators/management/configuration#port), then you must open the port specified for TCP.
  *   The only exception is for sequencers and transaction ingress nodes.
</Callout>

<Steps>
  ### Setup a Snap Sync Node

>   *   Expose op-geth [`--discovery.port`](/builders/node-operators/management/configuration#discoveryport) to the internet and figure out the default port and if there's any conflict with `op-geth`

Reviewer: ```suggestion
  *   Expose port `30303` (`op-geth`'s default discovery port) to the internet on TCP and UDP.
```

lang: en-US
description: Learn how to use and enable snap sync on your OP chain.
---

import { Callout, Steps } from 'nextra/components'

# Using Snap Sync for Chain Operators

This guide reviews the optional feature of Snap Sync for OP chains, including benefits and how to enable the feature.

Snap Sync significantly improves the experience of syncing an OP Stack node. Snap Sync is a native feature of go-ethereum that is now optionally enabled on `op-node` & `op-geth`.
Snap Sync works by downloading a snapshot of the state from other nodes on the network and is then able to start executing blocks from the completed state rather than having to re-execute every single block.
This means that performing a Snap Sync is significantly faster than performing a full sync.

*   Snap sync enables node operators on your network to sync faster.
*   Snap sync removes the need for nodes on your post Ecotone network to run a [blob archiver](/builders/node-operators/management/blobs).

## Enable Snap Sync for Chains

To enable snap sync, chain operators need to spin up a node which is exposed to the network and has transaction gossip disabled.

<Callout type="warning">

>   For snap sync, all `op-geth` nodes should be ran with the [`--discovery.port=30303`](/builders/node-operators/management/configuration#discoveryport) exposed to easily find other op-geth nodes to sync from.

Reviewer 2: I think we can remove `30303`, @trianglesphere will snap sync work on different ports?

  The only exception is for sequencers and transaction ingress nodes.
</Callout>

<Steps>
  ### Setup a Snap Sync Node

>   *   Expose op-geth [discovery port](/builders/node-operators/management/configuration#discoveryport) to the internet and figure out the default port and if there's any conflict with `op-geth`

Reviewer 2: Update here to the following:

Expose op-geth [--discovery.port](/builders/node-operators/management/configuration#discoveryport) to the internet.
